dapr/actor/runtime/stateprovider.py

```diff
@@ -78,14 +76,14 @@ async def save_state(
             json_output.write(operation)
             json_output.write(b'","request":{"key":"')
             json_output.write(state.state_name.encode('utf-8'))
+            json_output.write(b'"')
             if state.value is not None:
                 serialized = self._state_serializer.serialize(state.value)
```

Comment:
@artursouza As you pointed it out before, I changed the code to not encode the data in base64. Please double-check if it is expected or not.

cc/ @amanbha